docs/dotnet-getting-started.md

```diff
@@ -71,6 +71,8 @@ However, you will need to declare the MindeeClient in your Startup.cs or Program
 services.AddMindeeClient();
 ```
 
+(Or, you could also simply instanciate a new instance of `MindeeClient` but it will require to pass an instance of IConfiguration to get the Api Key.)
```

Comment:
```suggestion
(Or, you could also simply instantiate a new instance of `MindeeClient` but it will require to pass an instance of IConfiguration to get the Api Key.)
```